### ISSUE TITLE
Mac IPv6 SSDP not working

### ISSUE THREAD

**Issue by jeremyherbert** [JesusFreke/fusionIdea#14]
Related to https://github.com/JesusFreke/fusion_idea_addin/issues/1 perhaps?

When running, I get the following error in the pycharm event log:

```
java.lang.Throwable: Couldn't send ipv6 ssdp packet on any interface
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:143)
	at org.jf.fusionIdea.run.FusionScriptState$SSDPServer.sendIpv6SSDPRequest(FusionScriptState.java:268)
	at org.jf.fusionIdea.run.FusionScriptState$SSDPServer.lambda$start$0(FusionScriptState.java:283)
	at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
	at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:69)
	at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

The terminal shows:
```
Public key hash: F93552CFE1C6A07E748BE736BA257F74A94326CD
Server stopped.
Encountered error while attempting to connect to Fusion.
```

Pycharm and OS version info:
```
PyCharm 2020.2 (Professional Edition)
Build #PY-202.6397.98, built on July 28, 2020
Runtime version: 11.0.7+10-b944.20 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.15.4
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 16
Non-Bundled Plugins: io.protostuff.protostuff-jetbrains-plugin, intellij-micropython, org.jf.fusionIdea
```

**Comment by Woit** [comment]
Guys, any updates? @jeremyherbert any ideas pls?

**Comment by jphalip** [comment]
I realize this is an old thread, but just thought I'd point out a fix that I've been working on for this. See this [pull request](https://github.com/JesusFreke/fusionIdea/pull/22) and this related [discussion](https://github.com/JesusFreke/fusionIdea/issues/21).

**Comment by JesusFreke** [comment]
This should hopefully be fixed in v0.6.6, which I just released. Although it may take a day or two to become available on the intellij plugin repository
